services/workflows-service/src/metrics/service/metrics.service.ts

```diff
@@ -106,6 +106,26 @@ export class MetricsService {
           currentProjectId,
         ),
       },
+      cases: {
```

Comment:
I might not have understood it perfectly, but the numbers don't make sense to me - all in progress + all approved =/= all

Reply:
These are temporary mocks. The number will change soon.